templates/prune-job.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+/usr/local/bin/tarsnap --fsck --keyfile "{{ item.keypath }}" \
+--cachedir "{{ item.cache }}" && \
+tarsnapper --config /etc/tarsnapper.yml  expire dalite-logs "{{ item.name }}"
```

Comment:
Addressed in 65cd051621beae425e45f590c98438a8bf2ca41f . 

It was just copy-paste error. When I checked it worked, as the job I was checking was pruned.